content/en/serverless/insights/_index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Serverless Insights
+kind: documentation
+---
+
+{{< img src="serverless/serverless_actionable_insights_detail.png" alt="Auto-generated insights for your serverless applications" >}}
+
+Datadog automatically generates suggestions to resolve errors, performance problems, and optimize cost for your serverless applications.
```

Comment:
Missing a verb here: resolve errors, <verb?> performance problems, and optimize cost

Reply:
```suggestion
Datadog automatically generates suggestions to resolve errors and performance problems, and optimize cost for your serverless applications.
```
maybe?

Reply:
ha I just added the same exact suggestion as @agocs, so I'll delete mine and + to his